webapp2.py

```diff
@@ -211,6 +213,39 @@ def get(self, argument_name, default_value='', allow_multiple=False):
         else:
             return default_value
 
+    def get(self, argument_name, default_value='', allow_multiple=False):
+        """Returns the query or POST argument with the given name.
+
+        We parse the query string and POST payload lazily, so this will be a
+        slower operation on the first call.
+
+        :param argument_name:
+            The name of the query or POST argument.
+        :param default_value:
+            The value to return if the given argument is not present.
+        :param allow_multiple:
+            Return a list of values with the given name (deprecated).
+        :returns:
+            Return the value with the given name given in the request. If there
+            are multiple values, this will only return the first one. Use
+            `get_all()` to get multiple values.
+        """
+        return self._get(argument_name,
```

Comment:
Thanks for pushing back on this. I thought I remembered being admonished not to allow public methods in a class to call each other, because it could cause unpleasant surprises if one public method was overridden in a subclass, but I think that was more a "be careful when designing methods to be subclassed" warning.

In this case I agree that the redirection is not useful. Removing.